var/spack/repos/builtin/packages/py-kaleido/package.py

```diff
@@ -0,0 +1,18 @@
+# Copyright 2013-2023 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+class PyKaleido(PythonPackage):
+    """Static image export for web-based visualization libraries with zero dependencies"""
+
+    homepage = "https://github.com/wdecoster/nanostat"
+    pypi = "kaleido/kaleido-0.2.1.post1-py2.py3-none-manylinux2014_armv7l.whl"
```

Comment:
This wheel only works on Linux ARM. Can we download and build from GitHub source instead?

Reply:
I'm not entirely sure how to set that up. Can you send me a documentation link?

Reply:
Something along the lines of:
```python
url = "https://github.com/plotly/Kaleido/archive/refs/tags/v0.2.1.post1.tar.gz"

version(...)  # update the checksum

depends_on("py-setuptools", type="build")

build_directory = join_path("repos", "kaleido", "py")
```
See https://spack.readthedocs.io/en/latest/build_systems/pythonpackage.html and https://spack.readthedocs.io/en/latest/packaging_guide.html